.azure/hpu-tests.yml

```diff
@@ -25,9 +26,9 @@ pr:
       - "src/pytorch_lightning/**"
       - "tests/tests_pytorch/**"
       - "setup.cfg"  # includes pytest config
-      - ".actions/**"
+      - "setup.py"
     exclude:
-      - "requirements/**/docs.txt"
```

Comment:
I suggest we keep the old pattern in case deeper directories are added. Otherwise this change needs to be reflected checkgroup.yml too

Reply:
I don't agree with it but will I revert it and send in another PR

Reply:
ok, keeping one-level and if there is only one then set the name...